tests/test_dispatch_types.py

```diff
@@ -0,0 +1,160 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Tests for the frequenz.client.dispatch types."""
+
+from datetime import datetime, timedelta, timezone
+
+
+def test_component_selector() -> None:
+    """Test the component selector."""
+    # pylint: disable=import-outside-toplevel
+    from frequenz.client.common.microgrid.components.components import ComponentCategory
+
+    from frequenz.client.dispatch._types import (
+        ComponentSelector,
+        _component_selector_from_protobuf,
+        _component_selector_to_protobuf,
+    )
```

Comment:
Any reason for the imports inside the test?

Reply:
I guess keeping things near where they are used. No deeper reason

Reply:
If there is no good reason, I would put it on the top-level, which is the most popular style and doesn't require an exception for `pylint`.